commit 841abefc91ee870fe7cc8a4e36905b1655410c54
Author: Steve Dickson <steved@redhat.com>
Date:   Wed Aug 5 14:59:23 2020 -0400

    rpc.idmapd: Turn down the verbosity in flush_inotify()
    
    Commit 27a8e146 introduce a debugging message
    that was not cover by a check if verbose
    is set, which cause a large number of message
    to be logged on every kerberos mount
    
    Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1867172
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/idmapd/idmapd.c b/utils/idmapd/idmapd.c
index 8631414..7d1096d 100644
--- a/utils/idmapd/idmapd.c
+++ b/utils/idmapd/idmapd.c
@@ -500,7 +500,8 @@ flush_inotify(int fd)
 		     ptr += sizeof(struct inotify_event) + ev->len) {
 
 			ev = (const struct inotify_event *)ptr;
-			xlog_warn("pipefs inotify: wd=%i, mask=0x%08x, len=%i, name=%s",
+			if (verbose > 1)
+				xlog_warn("pipefs inotify: wd=%i, mask=0x%08x, len=%i, name=%s",
 				  ev->wd, ev->mask, ev->len, ev->len ? ev->name : "");
 		}
 	}
